### ISSUE TITLE
gc-ms xcms make parameters choosable

### ISSUE THREAD

**Issue by hechth** [RECETOX/workflow-testing#23]
Seems like if not explicitly instructed the parameters for the workflow are fixed - I think we should make the filter choice for XCMS etc. also available to change for users

**Comment by xtrojak** [comment]
Only parameters which do not change the number of inputs can be changed.

![image](https://user-images.githubusercontent.com/15349569/229149307-4e9bdcf4-35e1-4e96-b2ed-7fa6434ceeb1.png)

**Comment by hechth** [comment]
@xtrojak so this can be closed?

**Comment by hechth** [comment]
@mvdbeek can this somehow be implemented with the conditionals? Should be possible, right?

**Comment by mvdbeek** [comment]
Yes, it's a little clunky but you can include the step multiple times, then build a switch logic where only one of the conditional steps runs.
